docs/src/examples/components/Input/Variations/index.tsx

```diff
@@ -9,6 +9,11 @@ const Variations = () => (
       description="An input can have an icon."
       examplePath="components/Input/Variations/InputExampleIcon"
     />
+    <ComponentExample
+      title="Clearable icon"
+      description="An input can have a search icon that can change into clear button on typing."
```

Comment:
This feature is only showing that an input can make its value clearable.

The use of the search icon is unrelated and should not be in the description.  In fact, it may warrant a separate example showing that the clearable icon will override a normal icon when there is a value.  The base example doesn't actually require an existing icon in the input, correct?

Reply:
I'm trying to imagine the example where the input is clearable and doesn't have an icon. I might misunderstand the original ask...

Reply:
For example, I would expect as a user that I could do this:

```jsx
<Input clearable />
```

And it would result in an input that can clear its value.  As a user, I do not have to also supply an `icon='search'` prop in order for the clearable feature work.  I can supply one if I wish, and it will be replaced with an `x` when there is an input value, but it is not required for the `clearable` feature to work.

### SUIR Loading Input

Here's a corresponding example from SUIR.  An Input can be in the `loading` state.  The input can be defined with or without an `icon` prop.  Here is the same loading input, but shown with the icon defined:

```jsx
<Input placeholder='With an icon' icon='search' loading />
<Input placeholder='Without an icon' loading />
```

Here I am toggling the `loading` prop:

![http://g.recordit.co/TnO9WfAVBH.gif](http://g.recordit.co/TnO9WfAVBH.gif)

The `clearable` prop would work similar.  The use of the `icon='search'` is not required for the clearable icon to show up when there is a value.